packages/ui-components/Common/CircularIcon/index.tsx

```diff
@@ -0,0 +1,33 @@
+import classNames from 'classnames';
+
+import styles from './index.module.css';
+
+export type CircularIconProps = {
+  kind:
+    | 'event'
+    | 'method'
+    | 'property'
+    | 'class'
+    | 'module'
+    | 'classMethod'
+    | 'ctor';
```

Comment:
These kinds link up directly with the types in `HeadingMetadataEntry` of `api-docs-tooling`.